### ISSUE TITLE
Error: Cannot find module './patterns' - server not restarted after 3rd try

### ISSUE THREAD

**Issue by cacarr-pdxweb** [mvdschee/web-accessibility#20]
Open a directory:

```
Error: Cannot find module './patterns'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:602:15)
    at Function.Module._load (internal/modules/cjs/loader.js:528:25)
    at Module.require (internal/modules/cjs/loader.js:658:17)
    at require (internal/modules/cjs/helpers.js:20:18)
    at Object.<anonymous> (/home/pdxweb/.vscode-insiders/extensions/maxvanderschee.web-accessibility-0.2.4/server/out/server.js:15:17)
    at Object.<anonymous> (/home/pdxweb/.vscode-insiders/extensions/maxvanderschee.web-accessibility-0.2.4/server/out/server.js:205:3)
    at Module._compile (internal/modules/cjs/loader.js:711:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:722:10)
    at Module.load (internal/modules/cjs/loader.js:620:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:559:12)
[Info  - 11:52:50 PM] Connection to server got closed. Server will restart.
internal/modules/cjs/loader.js:604
    throw err;
    ^

Error: Cannot find module './patterns'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:602:15)
    at Function.Module._load (internal/modules/cjs/loader.js:528:25)
    at Module.require (internal/modules/cjs/loader.js:658:17)
    at require (internal/modules/cjs/helpers.js:20:18)
    at Object.<anonymous> (/home/pdxweb/.vscode-insiders/extensions/maxvanderschee.web-accessibility-0.2.4/server/out/server.js:15:17)
    at Object.<anonymous> (/home/pdxweb/.vscode-insiders/extensions/maxvanderschee.web-accessibility-0.2.4/server/out/server.js:205:3)
    at Module._compile (internal/modules/cjs/loader.js:711:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:722:10)
    at Module.load (internal/modules/cjs/loader.js:620:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:559:12)
[Info  - 11:52:50 PM] Connection to server got closed. Server will restart.
internal/modules/cjs/loader.js:604
    throw err;
    ^

Error: Cannot find module './patterns'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:602:15)
    at Function.Module._load (internal/modules/cjs/loader.js:528:25)
    at Module.require (internal/modules/cjs/loader.js:658:17)
    at require (internal/modules/cjs/helpers.js:20:18)
    at Object.<anonymous> (/home/pdxweb/.vscode-insiders/extensions/maxvanderschee.web-accessibility-0.2.4/server/out/server.js:15:17)
    at Object.<anonymous> (/home/pdxweb/.vscode-insiders/extensions/maxvanderschee.web-accessibility-0.2.4/server/out/server.js:205:3)
    at Module._compile (internal/modules/cjs/loader.js:711:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:722:10)
    at Module.load (internal/modules/cjs/loader.js:620:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:559:12)
[Error - 11:52:50 PM] Connection to server got closed. Server will not be restarted.

```

 ** VS Code version - 1.33.0-insider **

**Comment by mvdschee** [comment]
Thanks for the report, this has to do with the file starting with a Capital letter in GitHub, but in my local development environment starting with a no capital. For some reason, GitHub doesn't see that as a change. I will rename the file, and that should sort things out.

**Comment by mvdschee** [comment]
The issue has been resolved and is part of release 0.2.5
